### ISSUE TITLE
Declared variables in test are not in scope. 

### ISSUE THREAD

**Issue by luardo** [ui-jar/ui-jar#74]
I have my test files, and I want to inject some mockdata to my component using a factory which I have created. However the variable I create to contain the mock data is not being recognize in the styleguide. My test are running fine, and I only get this issue in uijar. 
This is my test
` /**@uijarexample */
it('should create', () => {
        const highlightsContainer = highlightsFactory.build();
        component.highlights = highlightsContainer.highlights;
...`

And this is the error I get 
`ERROR ReferenceError: highlightsContainer is not defined. 
`

Is there any work around to declare variable in the right scope? or is this a bug?

**Comment by nording** [comment]
Hm, sounds like a bug. Does it work of you do like this:
`component.highlights = highlightsFactory.build();` ?

**Comment by luardo** [comment]
Yes, when I do that it works. What I have seen in the generated code in the temp folder, is that the variables are not being copied or declared there. So it might be something related to the parser.

**Comment by nording** [comment]
Nice! I will merge your fix! Thanks!
